### PR TITLE
Fedora installation instructions

### PR DESCRIPTION
**Pull request by tim77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,14 @@ and the git version which builds from the most recent commit.
 Instruction of how to install AUR content can be found here:
 https://wiki.archlinux.org/index.php/Arch_User_Repository
 
+### Fedora and CentOS 7
+
+Packages for Fedora and CentOS 7 are available via [Copr](https://copr.fedorainfracloud.org/coprs/atim/lazygit/) (Cool Other Package Repo).
+
+```sh
+sudo dnf copr enable atim/lazygit -y && sudo dnf install lazygit -y
```
Comment (mjarkk) [inline README.md]: Maybe it's better to replace the ` && ` with a `\n`

**Comment by tim77** [comment]
> I think there was also already someone else who made a rpm package that worked on fedora as far as i know

Oh nice, would be nice to contact this maintainer and maybe we can package it properly for official Fedora/EPEL repos.

> But it might be better if we prefer copr over a custom repo.

TBH this my current build is far from perfect and i plan to package it properly in future for official repos. Copr is not official repos either, it is users repository, but Copr itself is part of official Fedora infrastructure, at least this is better then custom repos.

File: README.md (hunk 1 above)
Comment (tim77) [reply README.md]: I'll fix this in next commit...

**Comment by dawidd6** [comment]
Thanks!

**Pull request merged**